docs/proposals/docs-process.md

```diff
@@ -0,0 +1,94 @@
+# Docs Process
+
+@johndmulhausen
+
+A new design for the kubernetes.io site is currently being implemented and we think it's a good time to streamline and simplify the docs creation process as we rebuild the site from scratch.
+
+When a community is empowered to be effective contributors, great things happen. That’s why I’m proposing that we move the documentation from the docs/ folder in the core Kubernetes repo in GitHub into the GitHub pages repo itself, migrate that repo into one named `kubernetes.github.io` so that it will be compatible with the GitHub Pages auto-update system, and leave all munging scripts behind.
+
+**TL;DR: Right now we do a lot of scripting and branching to create docs -- this proposal is about using Jekyll's native capability and GitHub Page's staging capability to create a much easier-to-edit set of documentation that will ship with our upcoming new site.**
```

Comment:
The reason for a lot of the munging is that "markdown" is not a spec.  What GitHub itself allows and renders, Jekyll (well, kramdown and/or redcarpet) sometimes does not.  The mungers are hard-won lessons that tell me it is not practically possible to make humans write "correct" markdown without a syntax checker.  And if we check it, we might as well fix it if we can.  == mungers.  I haven't read this whole proposal yet, I just wanted to say that they exist for a reason :)